packages/terra-avatar/src/common/Avatar.module.scss

```diff
@@ -0,0 +1,178 @@
+@import '~terra-mixins/lib/Mixins';
+
+:local {
+  .avatar {
+    align-items: center;
+    background-clip: padding-box; // Stops background color from bleeding through border on IE.
+    border: var(--terra-avatar-v1-border, solid 1px #fff);
+    border-radius: 50%;
+    color: var(--terra-avatar-v1-color, #fff);
```

Comment:
In anticipation of the major version bump I think you'll want to set these all the `-v2-`. Find and replace should make this a pretty quick change.

Reply:
argggggggg